Jenkinsfile

```diff
@@ -1,13 +1,24 @@
 
 node {
   stage('checkout sources') {
-        // You should change this to be the appropriate thing
-        git url: 'https://github.com/jschmersal-cscc/special-topics-labs-ci'
+        // You should change this to be the appropriate thing \changed
+        git url: 'https://github.com/markdev8/special-topics-labs-ci'
+        //  git url: '/home/beachm4/special-topics-labs/special-topics-labs-ci'
+
   }
 
-  stage('Build') {
-    // you should build this repo with a maven build step here
-    echo "hello"
+  try {
+
+    stage('Build') {
+        echo 'Hello WORLD'
+        // you should build this repo with a maven build step here \snippet inserted
+        withMaven (maven: 'maven3') {
+            sh "mvn package"
+        }
+    }
+  } finally {
+    junit 'build/reports/**/*.xml'
```

Comment:
So this is trying to create a junit test results report page based on the results of running the tests, which is good.  However,  it's looking in the workspace for `build/reports/**/*.xml ` but that's not where maven puts its test output.   You'll want to change the path to be somewhere under `target`.  You can see where your build output goes when you run `mvn install ` to get a better idea.

Reply:
To expand on your question,  this will add a 'Test Results' link on your build results page when it's configured right